spec/install/.gitignore

```diff
@@ -1,5 +1,7 @@
 node_modules
 typings
 conf.js
+browserts_spec.js
```

Comment:
Yes. I have a `browserts_spec.ts` that transpiles to `browserts_spec.js`. I also have a similar test for javascript with `browserjs_spec.js`.